scss/_body.scss

```diff
@@ -0,0 +1,144 @@
+//ARTICLE BODY ELEMENTS
+.n-content-body {
+
+	// STANDARD TYPOGRAPHY AND ELEMENTS
+	@include oTypographySans(m);
+	margin-bottom: 40px;
+
+	&--serif {
+		@include oTypographySerif(m);
+	}
+
+	> p {
+		margin: 0.3em 0 0.8em;
+	}
+
+	> *:first-child {
+		margin-top: 0;
+	}
+
+	a {
+		@media print {
+			&:after {
+				text-decoration: none;
+				word-break: break-all;
+				word-wrap: break-word;
+			}
+			&[href^='/']:after {
+				content: ' (http://next.ft.com' attr(href) ')';
+			}
+			&:not([href^='/']):after {
+				content: ' (' attr(href) ')';
+			}
+		}
+	}
+
+	strong {
+		font-weight: 600;
+	}
+
+	// COPYRIGHT
+
+	.copyright-notice {
+		@include oTypographySansSize(s);
+		margin-top: 40px;
+		clear: both;
+	}
+
+	@media print {
+		&:after {
+			content: 'Print a single copy of this article for personal use. Contact us if you wish to print more to distribute to others. © The Financial Times Ltd.';
+			display: block;
+			border-top: 1px solid #000000;
+			padding-top: 5px;
+		}
+	}
+
+	// BLOCKQUOTES
+	blockquote {
+		color: getColor('cold-1');
+		position: relative;
+
+		p {
+			@include oTypographySerifDisplay(m);
+		}
+
+		.quote--full {
+			padding-top: 32px;
+			padding-bottom: 0;
+
+			p:first-child {
+				&:before {
+					@include nextIcon(pullquote, getColor('cold-1'), 26);
+					position: absolute;
+					top: 0;
+					left: 16px;
+					content: "";
+				}
+			}
+
+		}
+	}
+
+	// DATA TABLES
+	.data-table {
```

Comment:
data-table is a specific methode / capi thing that is different from a standard table, so would suggest sticking with data-table.